src/ui/public/vislib/lib/axis/axis_labels.js

```diff
@@ -123,10 +123,16 @@ export function VislibAxisLabelsProvider() {
       return function (selection) {
         selection.each(function () {
           selection.selectAll('text')
-          .attr('style', function () {
-            const currentStyle = d3.select(this).attr('style');
-            return `${currentStyle} font-size: ${config.get('labels.fontSize')};`;
-          });
+            .attr('style', function () {
+              const currentStyle = d3.select(this).attr('style');
+              return `${currentStyle} font-size: ${config.get('labels.fontSize')};`;
+            })
+            .text(function (d) {
+              // fixes issue with missing labels for range term agg on Y axis
```

Comment:
Do you know where this ` to ` comes from? How is it that the contents of this node are wrong the first time?

Perhaps we should simplify this and just do
```
.text(d => d));
```

Since d is always the correct content.

If not, I would add some more explanation as to why it is where this ` to ` is produced and why it is wrong in the first place.